src/pages/PublishedSchedule/PublishedSchedule.jsx

```diff
@@ -53,7 +55,7 @@ const PublishedSchedule = () => {
 
     };
     renderTable();
-  }, [currentUser, curSeason]);
+  }, [currentUser, curSeason, allSeasons]);
```

Comment:
⚠️ **[eslint]** <[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>React Hook useEffect has a missing dependency: 'selectedSeason'. Either include it or remove the dependency array.